src/mpi/comm/comm_dup.c

```diff
@@ -72,17 +72,13 @@ int MPIR_Comm_dup_impl(MPIR_Comm *comm_ptr, MPIR_Comm **newcomm_ptr)
 #undef FCNAME
 #define FCNAME MPL_QUOTE(FUNCNAME)
 /*@
-
```

Comment:
whitespace changes here are not related to the functionality change, so they shouldn't be included. see the "what makes a good commit" section here: https://wiki.mpich.org/mpich/index.php/Version_Control_Systems_101